docs/_layouts/default.html

```diff
@@ -18,21 +18,39 @@
     <meta name="theme-color" content="#157878">
     <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
     <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
-    <link rel="icon" href="/favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="{{ '/favicon.ico' | relative_url }}" type="image/x-icon" />
   </head>
-  <body>
+  <body class="container">
+    <nav class="sidebar">
+      <ul class="nav-list">
+        <li>
+          <span class="nav-header">Bolt Hands-on Lab</span>
```

Comment:
It'd be awesome if this included the logo somehow! Or maybe just add the logo above or below it?